docs/framework/wcf/feature-details/grouping-queued-messages-in-a-session.md

```diff
@@ -20,25 +20,25 @@ Windows Communication Foundation (WCF) provides a session that allows you to gro
   
 1. Define a service contract that requires a session. Do this with the <xref:System.ServiceModel.OperationContractAttribute> attribute and by specifying:  
   
-    ```  
+    ```csharp
     SessionMode=SessionMode.Required  
     ```  
   
 2. Mark the operations in the contract as one-way, because these methods do not return anything. This is done with the <xref:System.ServiceModel.OperationContractAttribute> attribute and by specifying:  
```

Comment:
```suggestion
2. Mark the operations in the contract as one-way, because these methods do not return anything. This is done with the <xref:System.ServiceModel.OperationContractAttribute> attribute by specifying:  
```